stubs/app/server/server.ts

```diff
@@ -31,7 +31,8 @@ if (!process.env.GRIST_SINGLE_ORG) {
 }
 
 setDefaultEnv('GRIST_UI_FEATURES', 'helpCenter,billing,templates,multiSite,multiAccounts,sendToDrive');
-
+setDefaultEnv('GRIST_WIDGET_LIST_URL',
+  'https://github.com/gristlabs/grist-widget/releases/download/latest/manifest.json');
```

Comment:
How about centralizing this constant in app/common/gristUrls.ts:commonUrls?
Apart from that, seems good.